packages/react-components/src/components/CameraButton.tsx

```diff
@@ -43,10 +44,8 @@ export interface CameraButtonProps extends ControlBarButtonProps {
   strings?: Partial<CameraButtonStrings>;
 }
 
-const onRenderCameraOnIcon = (): JSX.Element => <Video20Filled key={'videoIconKey'} primaryFill="currentColor" />;
-const onRenderCameraOffIcon = (): JSX.Element => (
-  <VideoOff20Filled key={'videoOffIconKey'} primaryFill="currentColor" />
-);
+const onRenderCameraOnIcon = (): JSX.Element => <Icon iconName="Video20Filled" />;
```

Comment:
iirc, The `primaryFill="currentColor"` attribute was added to avoid a warning from fluent about incorrect use of the `primaryFill` CSS property. You might want to keep that. (If you don't see any new warnings, you're good to go)

also, nit: Make this a componet? `DefaultCameraOnIcon` etc?

Reply:
James has fixed that in one of his PR and this is not needed anymore: https://github.com/Azure/communication-ui-library/pull/687

Reply:
@prprabhu-ms Regarding renaming it, this is throwaway code used only as sample ;)